AdminWebsite/AdminWebsite/ClientApp/src/app/common/constants.ts

```diff
@@ -10,7 +10,7 @@ export const Constants = {
     Representative: 'Representative',
     PleaseSelectPattern: '^((?!Please select).)*$',
     TextInputPattern: /^[^%{}~|]+$/,
-    TextInputPatternDisplayName: /^([-A-Za-z0-9 ',._])*$/,
+    TextInputPatternDisplayName: /^[\p{L}\p{N}\s]+$/u,
```

Comment:
you'll need to update the corresponding validator in Bookings API as well (`ParticipantRequestV2`). Otherwise the save will fail

be sure to check this works for the judge and joh as well

Reply:
also, I don't think this is the right expression. If you do a test you'll see it doesn't accept characters like hyphens

eg
First-Last

this expression should do the trick I think

^[\p{L}\p{N} ',._-]*$

Reply:
done